guides/common/modules/proc_synchronizing-ansible-collectios.adoc

```diff
@@ -0,0 +1,39 @@
+:_module-type: PROCEDURE
+
+[id="proc_synchronizing-ansible-collectios_{context}"]
+= Synchronizing Ansible Collectios
+
+[role="_abstract"]
+On {Project}, you can synchronize your Ansible Collections from Automation Hub, cloud.redhat.com, and other {Project} instances.
+Ansible Collections will appear on {Project} as a new repository type in the {ProjectWebUI} menu under *Content*.
+
+NOTE: Ansible Collections live in the Library environment and the lifecyrcle management is not supported.
+Capsules with Pulp 3 have collections support and will sync collection repositories if tied to the Library environment.
+
+.Procedure
+
+. In the {ProjectWebUI}, navigate to *Content* > *Products*.
+. Select the required product name.
+. In the *Products* window, select the name of a product that you want to create a repository for.
+. Click the *Repositories* tab, and then click *New Repository*.
+. In the *Name* field, enter a name for the repository.
++
+The *Label* field is populated automatically based on the name.
+. From the *Type* list, select *ansible collection*.
+. In the *Upstream URL* field, enter the URL for the upstream collections repository.
++
+The URL can be any Galaxy endpoint.
+. Optional: In the *Requirements.yml* field, you can specify the list of collections you want to sync from the endpoint, as well as their versions.
++
```

Comment:
It is optional, but we should mention that without it users will sync *everything* from the hub/gallaxy. Maybe some example of the yml would be nice should the link below change or something like that  (up to you).

Reply:
This is sort of what our users expect - all repositories behave the same - by default they sync all content.